README.md

```diff
@@ -12,15 +12,26 @@ NOTE: It is important to be using at least this version of `npm`
 because of a bug that removes `node_modules` that are installed from
 GitHub and doesn't reinstall them which breaks the build
 
+## Application Architecture
+The standard `docker-compose.yml` file brings up multiple containers
+
+- bPanel routing + static file server
+- bcoin bitcoin node/wallet
+- TLS terminating reverse proxy
+
+Some plugins require TLS to function properly
+
 ## Setup Your Environment With Docker
 This is primarily a setup for development purposes
 (though it could be used in production with some modification).
 
-To spin up your webapp, server, a bcoin node on regtest, and generate
+To spin up your webapp, reverse-proxy, server, a bcoin node on regtest, and generate
 50 regtest BTC for your primary wallet, clone & navigate to this repo then:
 1. Run `npm install` to create a secrets.env file.
-2. Run `docker-compose up -d` to start everything.
-3. Navigate to [localhost:5000](http://localhost:5000) to see your webapp.
+1. Run `docker-compose up -d` to start everything.
+1. Navigate to [localhost:5000](http://localhost:5000) to see your webapp.
+1. Or navigate to [https://localhost](https://localhost) to use TLS - you will have to choose to trust the certificate
+
```

Comment:
Markdown will auto render numbers in the correct order, so if you use all `1`s, you can reorder them more easily